great_expectations/execution_engine/execution_engine.py

```diff
@@ -189,6 +200,27 @@ class SplitDomainKwargs:
 
 
 class ExecutionEngine(ABC):
```

Comment:
@alexsherstinsky I believe adding the `@public_api` decorator here will allow these docs to be shown in the Netlify preview

Reply:
@anthonyburdi I was not sure -- I thought if the method is public we should declare as such.  Please provide guidance as to keep or remove the public API decorator.  Thank you!